<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonMenu</class>
 <widget class="QWidget" name="PersonMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidgetPerson">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSearch">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonAddNewPerson">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Person</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Of Birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Of Death:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelGender">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelAge">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelYearofBirth">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelYearofDeath">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonRemovePerson">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Selected Person</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonEditPerson">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Selected Person</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButtonShowTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
